src/app/perspective-status.component.ts

```diff
@@ -442,13 +453,22 @@ export class PerspectiveStatus implements OnChanges, AfterViewInit, AfterViewChe
         .map((tColor) => tColor.toRGBCSS());
   }
 
-  private interpolateColors(score: number): string {
+  interpolateColors(score: number): string {
     let scoreLowerIndex = Math.floor(score * 100);
```

Comment:
I think this would read cleaner as `... = Math.min(Math.floor(score * 100), this.gradientColorScale.length - 1)`; it clearly shows what you're trying to do and does so more concisely.

Reply:
Done (commit not updated yet, just doing this for bookkeeping)